CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
We must adopt the CNCF CoC, please copy the file from https://github.com/tikv/tikv/pull/5670

Reply:
Okay, I just linked to this anchor